bootstrap.go

```diff
@@ -104,7 +103,7 @@ func bootstrap(s Session) {
 	if err == nil {
 		// We have already finished bootstrap.
 		return
-	} else if !errors2.ErrorEqual(err, errors.ErrDatabaseNotExist) {
+	} else if !terror.Schema.Equal(err, terror.DatabaseNotExists) {
```

Comment:
It is not easy but it is more safe, different package may use the same error to return if there is no class code, this is unsafe.